runtime/drivers/postgres/postgres.go

```diff
@@ -15,6 +15,37 @@ import (
 
 func init() {
 	drivers.Register("postgres", driver{})
+	drivers.RegisterAsConnector("postgres", driver{})
+}
+
+var spec = drivers.Spec{
+	DisplayName: "Postgres",
+	Description: "Connect to Postgres.",
+	SourceProperties: []drivers.PropertySchema{
+		{
+			Key:         "sql",
+			Type:        drivers.StringPropertyType,
+			Required:    true,
+			DisplayName: "SQL",
+			Description: "Query to extract data from Postgres.",
+			Placeholder: "select * from table;",
+		},
+		{
+			Key:         "pg_database_url",
+			DisplayName: "Postgress Connection String",
+			Type:        drivers.StringPropertyType,
+			Required:    false,
+			Href:        "https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING",
+			Placeholder: "postgresql://user:password@localhost/mydb",
+			Hint:        "Either set this or configure PG_DATABASE_URL env variable.",
```

Comment:
Suggestion: "Either set this or pass `--env connectors.postgres.database_url=...` to `rill start`"